charts/rancher-csp-billing-adapter/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v2
+name: rancher-csp-billing-adapter
+description: Cloud Service Provider Marketplace Billing Adapter for Rancher.
+version: 999
+appVersion: 999
```

Comment:
Why does the chart and appVersion have 999 as their values? This isn't for the CRD but rather for the chart (i.e. package) that installs them.

Reply:
These are just placeholders for now. They will eventually get replace with the app version, whenever we have that infrastructure in place.